code/index.html

```diff
@@ -4,19 +4,117 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">  
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <a name="top"></a>
+
+    <header class="header">
+      <img class="logo" src="./logo.svg" alt="Technigo logo">
+      <h1>SUSTAINABILITY and DEVELOPMENT <em>NEWS</em></h1>
+      <nav class="all-nav">
+        <div class="site-nav">
+          <button class="nav-button">
+            <a class="nav-link" href="">About</a>
+          </button>
+          <button class="nav-button">
+            <a class="nav-link" href="">Read <i class="arrow down"></i></a>
+          </button>
+          <button class="nav-button">
+            <a class="nav-link" href="">Contact us</a>
+          </button>
+        </div>
+
+        <div class="media-nav">
+          <a class="facebook" href="https://en-gb.facebook.com/">
+            <img class="icon" src="./images/facebook.png" alt="Facebook icon">
+          </a>
+          <a class="twitter" href="https://twitter.com/">
+            <img class="icon" src="./images/twitter.png" alt="Twitter icon">
+          </a>
+          <a class="instagram" href="https://www.instagram.com/">
+            <img class="icon" src="./images/instagram.png" alt="instagram icon">
+          </a>
+        </div>
```

Comment:
add `target="_blank"` to anchor tags linking outside of the website to open the links in a new window.